distgen/generator.py

```diff
@@ -19,13 +19,16 @@ class Generator(object):
     pm_spc = None
 
     def __init__(self, global_jinja_args=None):
+        here = os.path.dirname(os.path.abspath(__file__))
         self.pm_cfg = PathManager(
-            [os.path.join(sys.prefix, "share", "distgen", "distconf")],
+            [os.path.join(here, "distconf"),
+             os.path.join(sys.prefix, "share", "distgen", "distconf")],
             envvar="DG_DISTCONFDIR"
         )
 
         self.pm_tpl = PathManager(
-            [os.path.join(sys.prefix, "share", "distgen", "templates")],
+            [os.path.join(here, "templates"),
+             os.path.join(sys.prefix, "share", "distgen", "templates")],
```

Comment:
seems like this will mixup local data with system data

Reply:
This problem has always been there to certain degree if you look at how `PathManager.get_path` works with the `envvar`. Local and system data would get mixed up if the envvar was specified. I don't think this is a huge issue. The only way this would manifest would be if the system installation had some files that the local one doesn't, which is not a huge deal for now IMO. I'd suggest fixing this later.

Reply:
Truth, let's go for it now... I noticed because I tried to solve this in #66 (I dropped the use of the ENV vars for testing purposes there).  Ack.